content/spin/v2/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -344,23 +333,23 @@ User: {SENTENCE}
 
 /// A Spin HTTP component that internally routes requests.
 #[http_component]
-fn handle_route(req: Request) -> Result<Response> {
+fn handle_route(req: Request) -> Response {
     let mut router = Router::new();
-    router.post("/api/sentiment-analysis", perform_sentiment_analysis);
     router.any("/api/*", not_found);
+    router.post("/api/sentiment-analysis", perform_sentiment_analysis);
```

Comment:
Does this compile?  I am bit surprised that `Request` is accepted for `Request<Json<SentimentAnalysisRequest>>` but if it works it works!

Reply:
I'm also confused, but it compoles and works. See here: https://github.com/fermyon/spin/issues/1973